flatpak/io.github.celluloid_player.Celluloid.json

```diff
@@ -0,0 +1,188 @@
+{
+    "app-id": "io.github.celluloid_player.Celluloid",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "master",
+    "sdk": "org.gnome.Sdk",
+    "command": "celluloid",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--device=all",
+        "--share=network",
+        "--socket=pulseaudio",
+        "--filesystem=xdg-pictures",
+        "--talk-name=org.gtk.vfs",
+        "--talk-name=org.gtk.vfs.*",
+        "--talk-name=org.gnome.SettingsDaemon.MediaKeys",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf",
+        "--own-name=org.mpris.MediaPlayer2.GnomeMpv.*",
+        "--env=LC_NUMERIC=C"
+    ],
+    "modules": [
+        "org.luajit.Luajit.json",
```

Comment:
Is there a reason why this is in a separate file?

Reply:
I originally did that to make it slightly easier to share around. Its pretty small and low maintainance to just inline it though.